tools/scripts/generate_constant_assets.py

```diff
@@ -0,0 +1,132 @@
+from pathlib import Path
+from typing import Dict
+
+from rotkehlchen.constants.resolver import strethaddress_to_identifier
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+from rotkehlchen.utils.misc import timestamp_to_date, ts_now
+
+
+class ContextManager():
+
```

Comment:
I think is clear what this does but could you add a comment here so if someone needs to look at it the thing gets easier?

Reply:
sure. Fixing some more stuff at the moment. Will add it with them.